CONTRIBUTING.md

```diff
@@ -85,13 +85,37 @@ pytest tests/unit
 
 ### Building & Testing Documentation
 
+#### Livereload w/automatic rebuild
+
 You can build & preview documentation locally using the following steps.
 
 Change into the doc directory:
 ```commandline
 cd doc
 ```
 
+Run the doc build and host a webserver to preview:
+```commandline
+make serve
+```
```

Comment:
Before re-installing the package, this is broken on my system after running `make clean`:

Before re-install:
```console
$ make serve
sphinx-autobuild --watch ../arcade -b html -d build/doctrees   . build/html
make: sphinx-autobuild: No such file or directory
make: *** [Makefile:62: serve] Error 127
```

Reply:
Is this because sphinx-autobuild was not installed, or some other issue?

For a first-time contributor, they already had to run `pip install -e .[dev]`, otherwise `sphinx-build` would not be installed.  So is it sufficient to rely on the existing setup instructions?

Reply:
I think it's because `make clean` deletes the `build` folder, but I can no longer replicate this with the latest changes after re-install, so I consider it resolved.